LiveSplit.AutoSplitters.xml

```diff
@@ -1,5 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <AutoSplitters>
+    <AutoSplitter>
+        <Games>
+            <Game>Gori: Cuddly Carnage</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/0kia/LiveSplit_AS/main/GoriAutoSplitter.asl</URL>
+            <URL>https://github.com/just-ero/asl-help/raw/main/lib/asl-help</URL>
+        </URLs>
```

Comment:
This autosplitter doesn't use `asl-help`. Why do you need it?

```suggestion
        <URLs>
            <URL>https://raw.githubusercontent.com/0kia/LiveSplit_AS/main/GoriAutoSplitter.asl</URL>
        </URLs>
```

Reply:
if i wanted to improve my script and use it in the future it would be easier to just have it there without having to re-pr. but i guess that's not an issue